src/evict/evict_lru.c

```diff
@@ -1421,7 +1625,6 @@ __evict_walk_file(WT_SESSION_IMPL *session, WT_EVICT_QUEUE *queue,
 
 		page = ref->page;
 		modified = __wt_page_is_modified(page);
-		page->evict_pass_gen = cache->evict_pass_gen;
```

Comment:
@fedorova This seems unrelated to your change.  This line was added fairly recently (Oct 21) and I think it likely that it was lost in a merge issue.  I'm going to restore it.  Speak up if you removed it intentionally at some point.